src/roiextractors/extraction_tools.py

```diff
@@ -59,44 +59,44 @@ class VideoStructure:
     consistency checks to ensure the validity of the sturcture.
 
     Attributes:
-        rows (int): The number of rows of each frame as a matrix.
-        columns (int): The number of columns of each frame as a matrix.
+        num_rows (int): The number of rows of each frame as a matrix.
+        num_columns (int): The number of columns of each frame as a matrix.
```

Comment:
Hmm... See other comment on `columns` vs `cols` - this case is at least inconsistent with the current state

Reply:
This escape me, I tried to change everything to cols.

Reply:
It might take a bit of iteration to purge the entire repo of the references, no prob